signer/src/config.rs

```diff
@@ -117,7 +120,7 @@ pub struct StacksNodeSettings {
     ///
     /// The endpoint to use when making requests to a stacks node.
     #[serde(deserialize_with = "url_deserializer")]
-    pub endpoint: url::Url,
+    pub endpoints: Vec<url::Url>,
```

Comment:
Yeah, split should always return at least one string. If it doesn’t convert to a URL then the function fails, otherwise we get a vec with at least one entry.

Reply:
Done